src/vs/platform/windows/common/windows.ts

```diff
@@ -151,7 +166,7 @@ export interface IWindowsService {
 	toggleSharedProcess(): TPromise<void>;
 
 	// Global methods
-	openWindow(paths: string[], options?: { forceNewWindow?: boolean, forceReuseWindow?: boolean, forceOpenWorkspaceAsFile?: boolean; }): TPromise<void>;
+	openWindow(paths: string[], options?: IOpenWindowOptions): TPromise<void>;
 	openNewWindow(): TPromise<void>;
 	showWindow(windowId: number): TPromise<void>;
 	getWindows(): TPromise<{ id: number; workspace?: IWorkspaceIdentifier; folderPath?: string; title: string; filename?: string; }[]>;
```

Comment:
Note, no functional changes in this file: I refactored the options to the interface and declared the base `IOpenOptions` interface to share between `IOpenConfiguration` and `IOpenBrowserWindowOptions`.